core/mutator/entry/mutation.go

```diff
@@ -28,10 +28,10 @@ import (
 type Mutation struct {
 	userID, appID string
 	index         []byte
-	data, nonce   []byte
+	Data, Nonce   []byte
```

Comment:
I prefer to keep these unexported and add getters.

Reply:
done